pkg/virt-api/api.go

```diff
@@ -635,18 +628,16 @@ func (app *virtAPIApp) composeSubresources() {
 	ws.Route(ws.GET("/stop-profiler").To(componentProfiler.HandleStopProfiler).Doc("stop profiler endpoint"))
 	ws.Route(ws.GET("/dump-profiler").To(componentProfiler.HandleDumpProfiler).Doc("dump profiler results endpoint"))
 
-	for _, version := range v1.SubresourceGroupVersions {
-		// K8s needs the ability to query info about a specific API group
-		ws.Route(ws.GET(definitions.GroupBasePath(version)).
-			Produces(restful.MIME_JSON).Writes(metav1.APIGroup{}).
-			To(func(request *restful.Request, response *restful.Response) {
-				response.WriteAsJson(subresourceAPIGroup())
-			}).
-			Operation(version.Version+"GetSubAPIGroup").
-			Doc("Get a KubeVirt API Group").
-			Returns(http.StatusOK, "OK", metav1.APIGroup{}).
-			Returns(http.StatusNotFound, httpStatusNotFoundMessage, ""))
-	}
+	// K8s needs the ability to query info about a specific API group
```

Comment:
This loop was not useless, it adds 2 endpoints:
```
/apis/subresources.kubevirt.io/v1
/apis/subresources.kubevirt.io/v1alpha3
```

This change removes the second endpoint. Is that what you want? If yes, can you explain in the commit message why?

Reply:
The loop/function adds the endpoint:

```
/apis/subresources.kubevirt.io
```

This function in the original loop reduces both versions to just the group:

https://github.com/kubevirt/kubevirt/blob/2ff23c535610edfb3df762cf68e52ebb652164ec/pkg/virt-api/definitions/definitions.go#L637

The endpoints you mention are added in the loop adding subresource endpoints for each version:

https://github.com/kubevirt/kubevirt/blob/2ff23c535610edfb3df762cf68e52ebb652164ec/pkg/virt-api/api.go#L510

Reply:
I didn't notice that. Then this change is ok.